base/file.jl

```diff
@@ -359,6 +359,12 @@ If `follow_symlinks=false`, and `src` is a symbolic link, `dst` will be created
 symbolic link. If `follow_symlinks=true` and `src` is a symbolic link, `dst` will be a copy
 of the file or directory `src` refers to.
 Return `dst`.
+!!! note
```

Comment:
You probably need a blank line here.

Reply:
Oops, missed that. Thanks!
Added a blank line. 👍🏼